waspc/src/Wasp/Analyzer/Parser/Lexer.x

```diff
@@ -97,6 +98,17 @@ alexGetByte (_, [], (currChar:remainingSource)) = case encodeChar currChar of
 alexInputPrevChar :: AlexInput -> Char
 alexInputPrevChar (prevChar, _, _) = prevChar
 
+-- | Tokenizes the given string. 
+alexScanTokens :: String -> Maybe [Parser Token]
+alexScanTokens str = go ('\n', [], str)
+  where
+    go input@(_, _, lexeme) = case alexScan input 0 of
```

Comment:
What is `lexeme` here? I thought it is rest of the source, not a lexeme that was tokenized? Yeah, I don't think this is lexeme. Lexeme, I believe, is actually what you get when you currently do `take len lexeme`.

Reply:
Don't remember why it's here, but definitely shouldn't be. I'll change it.